### PR TITLE
Add updating of distroless image to update_deps.sh

### PR DESCRIPTION
**Pull request by jacob-delgado**
Distroless images should be updated periodically. Add this capability to update_deps.sh so it gets updated.

Run UPDATE_BRANCH=master ./bin/update_deps.sh; make gen

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[X] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[X] Does not have any changes that may affect Istio users.

### CODE REVIEW

**Review by ericvn — APPROVED** [review]

File: docker/Dockerfile.distroless [hunk 1]
```diff
@@ -14,7 +14,7 @@ RUN echo isito-proxy:x:1337:1337:istio-proxy:/nonexistent:/sbin/nologin >> /home
 # - password file
 # - groups file
 # - /home/nonroot directory
-FROM gcr.io/distroless/static-debian10@sha256:5a4bd6a17f8d3c3e78df7ae62a95809e705c7315bccf59091af04b7b554cd037
+FROM gcr.io/distroless/static-debian10@sha256:34cb7dca899f9f40f576509977cd43b093cf13eeec2ce6333f9b9b2db331f937
```
Comment (howardjohn) [inline docker/Dockerfile.distroless:17]: Note we also ahve gcr.io/istio-release/iptables which is separate

Comment (howardjohn) [reply docker/Dockerfile.distroless:17]: and not on debian10

Comment (jacob-delgado) [reply docker/Dockerfile.distroless:17]: Added a update for proxyv2 as well

**Review by howardjohn — APPROVED** [review]

**Pull request merged**